linode/manager
# upcoming: [M3-7632] - Use infinite query for child accounts

## Description 📝
Considering parent accounts may have a large number of children, for better performance we're loading and displaying the number of child accounts by pages, which will update upon scrolling.

## Changes  🔄
List any change relevant to the reviewer.
- Updated MSW to account for changes
- Added new: `useChildAccountsInfiniteQuery`

## Preview 📷
**Include a screenshot or screen recording of the change**
![Screenshot 2024-02-12 at 1 12 08 PM](https://github.com/linode/manager/assets/125309814/7f9c9cd6-c8c6-4add-8956-bcde9f0d4908)


## How to test 🧪

### Prerequisites
(How to setup test environment)
- Turn on MSW

### Verification steps 
(How to verify changes)
- Open user menu and click "Switch Account"
- Observe first 25 accounts load and rest are fetched upon scroll

## As an Author I have considered 🤔

*Check all that apply*
